tests/test_dashboards.py

```diff
@@ -16,6 +16,10 @@ def setUpClass(cls) -> None:
     def tearDownClass(cls) -> None:
         id_ = cls.api.get_dashboard_id('pylana-dashboard')
         cls.api.delete_dashboard(id_)
+
+        dashboard_id = cls.api.get_dashboard_id('incident-test-dashboard')
+        cls.api.unshare_dashboard(dashboard_id,
+                                  [cls.api.user.organization_id]).json()
```

Comment:
It is good to denote discarded return values with an underscore `_ = ...`. Also, we don't need the `.json()` here.